app/helpers/external_resources_form_helper.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+module ExternalResourcesFormHelper
+  include FormHelper
+
+  def new_external_resource_props
+    new_form_props(external_resources_form_inputs, resources_path)
+  end
+
+  private
+
+  def external_resource_input_props(field, type, label, group = false)
+    { id: "external_resource_#{field}", type: type,
+      name: "external_resource[#{field}]#{group ? '[]' : ''}",
+      label: label }
```

Comment:
Tou can format it like you did in `external_resource_languages`, using one key per line.

```suggestion
    {
      id: "external_resource_#{field}",
      type: type,
     name: "external_resource[#{field}]#{group ? '[]' : ''}",
     label: label
    }
```